src/latch_cli/services/launch_v2/interface.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any, Optional
+
+import requests
+from requests.adapters import HTTPAdapter
+from urllib3.util.retry import Retry  # type: ignore  # noqa: PGH003
+
+from latch.utils import current_workspace
+from latch_sdk_config.latch import config
+
+session = requests.Session()
+retries = 5
+retry = Retry(
+    total=retries,
+    read=retries,
+    connect=retries,
+    method_whitelist=False,
+)
+adapter = HTTPAdapter(max_retries=retry)
+session.mount("http://", adapter)
+session.mount("https://", adapter)
+
+
+def get_workflow_interface(
+    token: str, wf_name: str, version: Optional[str] = None
+) -> tuple[int, dict[str, Any], dict[str, Any]]:
+    """Retrieves the set of idl parameter values for a given workflow by name.
+
+    Returns workflow id, interface, and default parameters.
+    """
+
+    response = session.post(config.api.workflow.interface, headers={"Authorization": f"Bearer {token}"}, json={
+        "workflow_name": wf_name,
+        "version": version,
+        "ws_account_id": current_workspace(),
```

Comment:
pass in workspace from above and use `current_workspace` as the default if none is provided